qtpy/compat.py

```diff
@@ -129,3 +129,15 @@ def getsavefilename(parent=None, caption='', basedir='', filters='',
                                 caption=caption, basedir=basedir,
                                 filters=filters, selectedfilter=selectedfilter,
                                 options=options)
+
+# =============================================================================
+def imagebytes(qimage):
```

Comment:
```suggestion
def getimagebytes(qimage):
```

Unless there's a particular reason for this exact name, what about naming this function as a verb, per standard Python/PEP 8 convention as well as that used within this module?

Reply:
Oversight on my part, I've made this change and the function is now called 'getimagebytes'.